theme/login/messages/messages_en.properties

```
@@ -1,2 +1,2 @@
 loginAccountTitle=Sign in with VSHN Account or external provider.
-identity-provider-login-label=First time here? Just login with an external provider of your choice. For more details see https://kb.vshn.net/kb/vshn-login.
+identity-provider-login-label=First time here? Just login with an external provider of your choice. For more details see https://kb.vshn.ch/kb/vshn-login.html.
```

```suggestion
identity-provider-login-label=First time here? Just login with an external provider of your choice. See https://kb.vshn.ch/kb/vshn-login.html for more details.
```

Reword to mitigate the issue with regex also pulling the `.` into the link.